### PR TITLE
refactor(tooltip): 12087 - replace tooltip view with ui kit component

### PR DESCRIPTION
**Pull request by Akiyamka**
This mr just replace tooltip view ui-kit component

### CODE REVIEW

File: src/features/tooltip/components/PopupTooltip/PopupTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      tooltip?.onOuterClick?.(e, resetCurrentTooltip);
+    },
+    [resetCurrentTooltip, tooltip],
+  );
+
+  if (!tooltip) return null;
+
+  return (
+    <CSSTransitionWrapper in={Boolean(tooltip)} timeout={300} classNames={fadeClassNames}>
```
Comment (propakov) [inline src/features/tooltip/components/PopupTooltip/PopupTooltip.tsx:35]: fading animation stopped working locally

Comment (propakov) [reply src/features/tooltip/components/PopupTooltip/PopupTooltip.tsx:35]: found a way of how to fix the animation problem 
Screenshot 2023-01-13 at 11 17 04

but maybe there is a better way 🤷‍♂️

**Review by propakov — APPROVED** [review]
Apart from animation problem, code LGTM. So need to decide what to do with animation only  🚀

File: src/features/tooltip/components/PopupTooltip/PopupTooltip.tsx (hunk 1 above)
Comment (Akiyamka) [reply src/features/tooltip/components/PopupTooltip/PopupTooltip.tsx:35]: cool, nice catch, thank you very much for investigation!

**Pull request merged**